charts/andi/values.yaml

```diff
@@ -108,6 +108,10 @@ theme:
   secondaryColor: "#1176D4"
   successColor: "#008000"
   errorColor: "#B80000"
+  # base64 string for the favicon (optional).  If used, it is a base64-encoded gzipped version of the icon
+  # For instance:
+  # customFavIconBase64: "H4sICJYyW2QCA2Zhdmljb24uaWNvAGNgYARCAQEBBhDQYGRgEAPRQAwSUQBiRgYWsFwDAwIIILE36Gsx/G9hYNjcHs5AZaAAoZhQuIxKINKAgQnMNTZQgkgaG0BoJWNjCIOZ2YCgBf//U4YBodIDmz4BAAA="
+  customFavIconBase64: 
```

Comment:
could we keep the naming consistent for both of the fav icon variables so it's easier to search later